src/main/java/lotto/service/LottoService.java

```diff
@@ -0,0 +1,105 @@
+package lotto.service;
+
+import lotto.model.Lotto;
+import lotto.model.Rank;
+import lotto.util.ResultPrinter;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoService {
+
+    public void playLotto(int purchaseAmount, List<Integer> winningNumbers, int bonusNumber) {
+        validateAmount(purchaseAmount);
+        validateWinningNumbers(winningNumbers);
+        validateBonusNumber(bonusNumber, winningNumbers);
+
+        List<Lotto> lottoTickets = generateLottoTickets(purchaseAmount);
+        ResultPrinter.printLottoTickets(lottoTickets);
+
+        Map<Rank, Integer> result = calculateResults(lottoTickets, winningNumbers, bonusNumber);
+        ResultPrinter.printResults(result);
+
+        int totalPrize = calculateTotalPrize(result);
+        double profitRate = calculateProfitRate(totalPrize, purchaseAmount);
+        ResultPrinter.printProfitRate(profitRate);
+    }
+
+    private List<Lotto> generateLottoTickets(int purchaseAmount) {
+        int ticketCount = purchaseAmount / 1000;
+        List<Lotto> lottoTickets = new ArrayList<>();
+
+        for (int i = 0; i < ticketCount; i++) {
+            lottoTickets.add(Lotto.generate());
+        }
```

Comment:
의미를 알릴 수 없는 `i` 대신 다른 변수명을 사용하는 것은 어떨까요?
```suggestion
        for (int count = 0; i < ticketCount; count++) {
            lottoTickets.add(Lotto.generate());
        }
        for (int index = 0; i < ticketCount; index++) {
            lottoTickets.add(Lotto.generate());
        }
```

Reply:
앗 훨씬 좋은 것 같습니다! i는 코드를 읽는 입장에서 이해하기가 어려울 것 같습니다! 제안해주셔서 감사합니다! :)